src/Common/Suggest.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#include "ConnectionParameters.h"
+#include "../../programs/client/ConnectionParameters.h"
```

Comment:
Maybe it should be moved into common too?
Also this can be moved into .cpp, forward declaration of `ConnectionParameters` is enough here.